components/open-source-tools/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,16 @@
+= Open Source Tools
+
+List of open source tools supported by OpenZeppelin and their documentation.
```

Comment:
Perhaps this would make it clearer that this is a specific collection, rather than a comprehensive list:
```suggestion
A collection of open source tools supported by OpenZeppelin and their documentation.
```